js1-week-2/0.md

```diff
@@ -26,3 +26,6 @@ c) Try to summarise the main difference between `logSum` and `calculateSum`
 
 
 {YOUR ANSWERS HERE}
+
+a) `result1` will evaluate to (42) but the first function will not print anything .
```

Comment:
The result of the first one will be 42 because it is a function.
The second would be error because it is not a function.